spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceConfiguration.java

```diff
@@ -83,6 +85,9 @@
 	@Autowired
 	private MessageBuilderFactory messageBuilderFactory;
 
+	@Lazy @Autowired(required = false)
```

Comment:
Yes, it seems like we're triggering the early initialization of an entire tree of beans, and the postprocessor that installs the tuple and json property accessor doesn't run when it should. Leaving it as is, but we should clean up `ChannelBindingServiceConfiguration` before GA.